cross/pip/Makefile

```diff
@@ -1,6 +1,6 @@
 PKG_NAME = pip
 # use the same version in native/python3/Makefile (python3_native_post_install)
```

Comment:
pip version should be the same in native/python310/Makefile and native/python311/Makefile

Reply:
good catch, thnx.  done

Reply:
can you adjust the comment in the line above too?

Reply:
done